documentation/modules/auxiliary/scanner/http/http_traversal.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+This module allows you to test if a web server (or web application) is vulnerable to directory traversal.
+
+### Setup
+
+1. Install php and apache on your machine
+1. Remove everything in /var/www/html/
+1. Create `/var/www/html/index.php` with the following contents
+    ```
+    <?php
+    $p = 'home.php';
+    if (isset($_GET['p']))
+        $p = $_GET['p'];
+    include ($p);
+    ?>
+    ```
+1. Create `/var/www/html/home.php` with the following contents
+    ```
+    <!DOCTYPE html>
+    <html>
+    <head>
+        <title>Hello, World!</title>
+    </head>
+    <body>
+        <a href="?p=home.php">home</a>
+    </body>
+    </html>
+    ```
+1. Run the following command: `sudo systemctl start apache2.service`
+
+## Verification Steps
+
+1. Install the application
+1. Start msfconsole
+1. Do: `use auxiliary/scanner/http/http_traversal`
+1. Do: `set rhosts <rhost>`
+1. Do: `set path <filepath>`
+1. Do: `run`
+
+## Options
+
+1. DATA - HTTP body data
+1. DEPTH - Traversal depth (default: 5)
+1. FILELIST - file containing list of files to bruteforce for (default: /usr/share/metasploit-framework/data/wordlists/sensitive_files.txt)
+1. METHOD - HTTP request method to use (default: GET)
+1. PATH - Vulnerable path. Ex: /foo/index.php?pg= (default: /)
+1. PATTERN - Regexp pattern to determine directory traversal (default: ^HTTP/\d\.\d 200)
```

Comment:
Please change the options to follow the format outlined here:
https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md

Reply:
Fixed the format